app/models/activity.rb

```diff
@@ -79,7 +79,7 @@ class Activity < ApplicationRecord
   validates :policy_marker_disability, presence: true, on: :policy_markers_step, if: :requires_policy_markers?
   validates :policy_marker_disaster_risk_reduction, presence: true, on: :policy_markers_step, if: :requires_policy_markers?
   validates :policy_marker_nutrition, presence: true, on: :policy_markers_step, if: :requires_policy_markers?
-  validates :gcrf_challenge_area, presence: true, on: :gcrf_challenge_area_step, if: :is_gcrf_funded?
+  validates :gcrf_challenge_area, presence: true, on: :gcrf_challenge_area_step, if: -> { is_gcrf_funded? && !fund? }
```

Comment:
Would it make sense to make `!fund?` part of the `is_gcrf_funded?` method? Or should that method also return `true` for the GCRF fund itself?

Reply:
I kept them separate to avoid having to refactor [these tests](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/760/files/144a311195f54c6c6b3c4cef19d7d3502898ab9a#diff-6b85c30d286a1b9fda4df24c1e2ed295ebcc21cd045ad03fc96939d9cf51a25dR273-R278), but perhaps I should tidy those up instead.

Reply:
Ended up fixing the tests, and everything feels more logical now, so thanks 👍